schema_validation.py

```
@@ -0,0 +1,279 @@
+        if r is None:
+            print("FAIL (No response object)")
+            return False
+
+        if r.status_code not in (200, 202):
+            print(f"FAIL (HTTP {r.status_code}): {r.text}")
+            return False
+
+        data = r.json()
+        model.model_validate(data)
+        print("PASS")
+        return True
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)